### DISCUSSION TITLE
Add the German DWD as an API source for weather information

### DISCUSSION THREAD

**Discussion by LeChrisDE** [breezy-weather/breezy-weather#387] [Ideas]
Hello everyone,

at first, the breezy-weather app has impressed me so much. It offers a very well-designed UI - much better as most of the (also commercial) competitors.

I'd like to suggest to add another source for the weather API - the official API of the "Deutscher Wetterdienst (DWD)" for Germany. Here is the description: https://dwd.api.bund.dev/

Thank you for your work and (if you like) the support and development including this further API!

Best regards,
Chris

**Comment by papjul** [comment]
Hello, it seems to me they don’t have an API by coordinates, only stations. You usually want weather data tailored for where you live, not from the station.

If you can find an API like that, you can create a new Issue, and use the "New source" template.

**Reply by LeChrisDE** [reply]
Hello and thank you for your quick response. 

I can ask via email for such an service - maybe they have it (for example for their own app) and don't want to offer it for public use - but I don't know this for sure. I will answer it here, if I get one.

**Reply by papjul** [reply]
Environment Canada has endpoints for cities elsewhere from the documentation for stations. If you know another weather app that has support for DWD, I can look into it. From URLs I can find inside this app, I’m sure I can find some documentation.

**Reply by LeChrisDE** [reply]
Sorry for my late answer:

But meanwhile The DWD has answered my questions to the mentioned API:

The problem is, that the link in my first post is only the unofficial documentation by a third party - collecting many APIs in Germany and let the web site looks like an official collection. 

The API itself is an internal API which only is made for using it with their own "Warnwetter"-App. At this time there is no official API from the DWD for other weather apps like breezy-weather. But I hope the wish for such an interface is received and will come in future.

**Comment by Cod3dDOT** [comment]
https://brightsky.dev/ "JSON API for DWD open weather data"

**Reply by papjul** [reply]
If it's just the Open-Data part (ICON model, from my understanding), Open-Meteo already provides it seamlessly with other models when data is missing.

**Reply by papjul** [reply]
I implemented it for next version :)

**Reply by dktzde** [reply]
@papjul Thanks a lot. Now I undersand, why I cannot find "Bright Sky" even if I seen it at [sources.md](https://github.com/breezy-weather/breezy-weather/blob/main/docs/SOURCES.md)
